src/main/java/server/Server.java

```diff
@@ -2,17 +2,17 @@
 
 import java.io.*;
 import java.net.*;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.nio.file.FileSystems;
+import java.nio.file.*;
 import java.util.ArrayList;
 import java.util.Scanner;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import org.apache.tomcat.util.http.fileupload.FileUtils;
```

Comment:
Is this a new dependency?

Reply:
Yes, need it to copy files from one folder to another (for `configurations`) can't do that with Java std lib.

Reply:
This looks simple enough? https://www.callicoder.com/java-copy-directory-recursively/

Reply:
Of course, you can write this a bit more imperatively if you don't like the functional style.

Reply:
Well, in general, more dependencies make the code more complicated. Of course, there are plenty of good reasons not to re-invent the wheel every time which is why we're not writing assembly anymore.

The question then is: When is it worth it? Maybe I'm too much of a C/C++ dev to talk about this, but some elementary tasks can be done just as well by using the standard library. What I'm scared about here most is that I don't know how this is being handled. You've added the import but as far as I can tell you haven't touched the build files; yet the CI passes. How does this work? Has this already been added? If so, there's no problem whatsoever. Otherwise, we're relying on some behaviour I quite frankly don't understand.

Reply:
Just realised the `org.apache.tomcat.util.http.fileupload.FileUtils` import you're referring to is actually unused. Deleted line.